Change "System Report?" to "Visible for other users" (#272)
mautic/user-documentation: docs/reports/reports.rst

  
* **Published** - Select **Yes** to ensure that the Report is active, ready to send when scheduled in the Report options.

* **Visible for all logged-in Users** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.

> * **Owner** - Select the owner of the Report to determine who can see the Report if the 'Visible for all logged-in users' setting is No.

Reviewer: ```suggestion
* **Owner** - Select the owner of the Report to determine who can see the Report if the 'Visible for all logged-in Users' setting is No.
```
Looks like this one needs changing too according to Vale!

  
* **Published** - Select **Yes** to ensure that the Report is active, ready to send when scheduled in the Report options.

> * **Visible for all logged-in users** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.

Reviewer: ```suggestion
* **Visible for all logged-in Users** - Select **Yes** so that all Users can access the Report. If set to No, only the owner of the Report and Users with permission to view others' Reports can see the Report.
```

Author: Is this a standard?
Or we are trying to make it standard?

Reviewer: In the docs we always capitalise features of the product - we apply the Google Technical Writing Style Guide: https://developers.google.com/style - although it seems there was a recent update which now says _not_ to capitalise features where it used to say that you should!

For the time being we stick with what we currently use.

> * **Owner** - Select the owner of the Report to determine who can see the Report if the Visible for all logged-in users setting is No.

Reviewer: ```suggestion
* **Owner** - Select the owner of the Report to determine who can see the Report if the 'Visible for all logged-in users' setting is No.
```
